libbeat/processors/communityid/communityid.go

```diff
@@ -211,43 +213,31 @@ func tryToIP(from interface{}) (net.IP, bool) {
 	}
 }
 
-// tryToUint16 tries to coerce the given interface to an uint16. On success it
+// tryToUint tries to coerce the given interface to an uint16. On success it
 // returns the int value and true.
-func tryToUint16(from interface{}) (uint16, bool) {
+func tryToUint(from interface{}) (uint, bool) {
 	switch v := from.(type) {
 	case int:
-		return uint16(v), true
+		return uint(v), true
 	case int8:
-		return uint16(v), true
-	case int16:
-		return uint16(v), true
-	case int32:
-		return uint16(v), true
-	case int64:
-		return uint16(v), true
+		return uint(v), true
 	case uint:
-		return uint16(v), true
+		return uint(v), true
 	case uint8:
-		return uint16(v), true
-	case uint16:
-		return v, true
-	case uint32:
-		return uint16(v), true
-	case uint64:
-		return uint16(v), true
+		return uint(v), true
```

Comment:
```suggestion
		return uint(v), true
	case uint16:
		return uint(v), true
	case uint32:
		return uint(v), true
	case uint64:
		return uint(v), true
```

Reply:
made some changes and it looks like everything is working now.